src/main/scala/nl.knaw.dans.easy.deposit/authentication/ServletEnhancedLogging.scala

```diff
@@ -15,42 +15,75 @@
  */
 package nl.knaw.dans.easy.deposit.authentication
 
-import javax.servlet.http.HttpServletRequest
+import javax.servlet.http.{ HttpServletRequest, HttpServletResponse }
 import nl.knaw.dans.lib.logging.DebugEnhancedLogging
 import org.scalatra.{ ActionResult, ScalatraBase }
 
 import scala.util.{ Failure, Success, Try }
 
-// TODO candidate for dans-scala-lib (another package than authentication?)
+// TODO candidate for dans-scala-lib / another package than authentication?
 trait ServletEnhancedLogging extends DebugEnhancedLogging {
   this: ScalatraBase =>
 
   before() {
-    logger.info(s"${ request.getMethod } ${ request.getRequestURL } remote=${ request.getRemoteAddr } params=$params headers=${ request.headers }")
+    // TODO a library version should filter any authentication any client might invent and probably provide hooks for more filters
```

Comment:
So far this covers all the types of authentication we support as far as I know.

The any-any is kind of an open end. It might be a good idea (but beyond the scope of this PR) to protect end users that try other common authentication methods, and be safe in case we decide to implement other authentication methods and forget to update the logging-library.

@rvanheest you introduced the concept of hooks in this context. What are you thinking of? I don't yet know of a simple hook for the before method, below a simple hook for the responses (requires to add `()` to the current calls).

```suggestion
    def logResponse(msg: String="")(implicit request: HttpServletRequest, response: HttpServletResponse): ActionResult = {
      logger.info(s"${ request.getMethod } returned status=${ actionResult.status } $msg; authHeaders=$authHeaders; actionHeaders=${ actionResult.headers }")
```